core/src/main/java/com/linecorp/armeria/client/ClientFactory.java

```diff
@@ -61,7 +61,14 @@
     /**
      * The default {@link ClientFactory} implementation.
      */
-    ClientFactory DEFAULT = new ClientFactoryBuilder().build();
+    ClientFactory DEFAULT = builder().build();
```

Comment:
Should we switch to `ofDefault()`?

Reply:
+1 I would lean towards using `ofDefaults()`

Reply:
OK. Let me take care of it in another pull request.